docs/demos/onvif-authentication-access.md

```diff
@@ -0,0 +1,306 @@
+### Passing credentials to Onvif Discovery Handler to discover authenticated devices
+Make sure you have at least one Onvif camera that is reachable so Onvif discovery handler can discovery your Onvif camera. To test accessing Onvif with credentials, make sure your Onvif camera is authentication-enabled. **Write down the username and password**, they are required in the flow below.
+
+#### Acquire Onvif camera's device uuid
```

Comment:
Maybe add a note that if you already know your device UUID you can skip this step

Reply:
add a note